redesign-project/src/styles.scss

```diff
@@ -0,0 +1,97 @@
+/* For desktop: */
+body {
+    background-color: #62718C
+}
+
+.background {
+    display: none;
+}
+
+.desktop_background {
+    width: 96%;
+    position: absolute;
+    margin-left: 20px;
+    margin-top: 175px;
+}
+
+.rectangle {
+    width: 400px;
+    height: 258px;
+    border-radius: 25px;
+    background-color: white;
+    position: absolute;
+    margin-top: 24px;
+    margin-left: 60px;
+}
+
+.header {
+    width: 370px;
+    position: absolute;
+    margin-top: 40px;
+    margin-left: 75px;
+}
+
+
+.rectangle_large {
+    width: 96%;
+    height: 875px;
+    background-color: white;
+    position: absolute;
+    margin-top: 893px;
+    margin-left: 20px;
+    margin-bottom: 40px;
+}
+
+.footer {
+    display: none;
+}
+
+
+
+/* For mobile: */
+@media only screen and (max-width: 326px) {
+
+    body {
+        background-color: #62718C
+    }
+
+    .background {
+        width: 90%;
+        margin-top: 40px;
+        margin-left: 2.5%;
+        position: absolute;
+    }
+
+    .rectangle {
+        width: 71%;
+        height: 150px;
+        border-radius: 25px;
+        background-color: white;
+        position: absolute;
+        margin-left: 12%;
+    }
+
+    .header {
+        width: 65%;
+        position: absolute;
+        margin-top: 10px;
+        margin-left: 15%;
+    }
+
+    .footer {
+        width: 35px;
+        margin-top: 392px;
+    }
+
+    .footer__insta {
```

Comment:
For the social icons, maybe a more BEM-like name would be `footer__icons--insta`? Footer is the block-level container, icons are the elements inside that container and the social media networks (insta, fb) are the modifiers.